### PR TITLE
feat: add Bahasa Indonesia

### PR DESCRIPTION
**Pull request by adityapurwa**
# Context

Adding Bahasa Indonesia (id-ID) language localization.

### CODE REVIEW

File: website/next-i18next.config.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ module.exports = {
       "vi",
       "zh",
       "tr",
+      "id-ID",
```
Comment (fozziethebeat) [inline website/next-i18next.config.js:23]: Other locales are using just the language code.  How essential is the country code here?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
If possible, I'd prefer we use just `id` for the locale value and directory name.

File: website/next-i18next.config.js (hunk 1 above)
Comment (adityapurwa) [reply website/next-i18next.config.js:23]: Thank you for the feedback! After a second thought, you are right. We don't need the country ID here. I updated the PR to only use language code.

Comment (fozziethebeat) [reply website/next-i18next.config.js:23]: Looks like this array entry needs to be updated as well to be `id`.

**Review by fozziethebeat — CHANGES_REQUESTED** [review]

**Comment by AbdBarho** [comment]
Blocked by #1327

File: website/public/locales/id/side_menu.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (fozziethebeat) [inline website/public/locales/id/side_menu.json:1]: We had some refactoring in another PR.  Sorry for another ask but can you move all these entries in to `common.json`?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]

File: website/public/locales/id/side_menu.json (hunk 2 above)
Comment (adityapurwa) [reply website/public/locales/id/side_menu.json]: Thank you for letting me know about the refactor. I moved these into common file in [65e210a](https://github.com/LAION-AI/Open-Assistant/pull/1337/commits/65e210a0e2a93c15e854704971fa8204e4e6e94f) 👍

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**